Task1/intro.json

```diff
@@ -174,12 +174,6 @@
     "hacktoberfest_round": "first",
     "role": ["contributor"]
   },
-  {

```

Comment:
Can you please put this back in the code?
Or update your branch?

Reply:
sure